wix/Main.wxs.template

```diff
@@ -1,238 +1,233 @@
-        <!-- skip the page on the way back too -->
-        <Publish Dialog="CustomizeDlg" Control="Back" Event="NewDialog" Value="WelcomeDlg"  Order="3">1</Publish>
-      <?endif?>
-    </UI>
-  </Product>
-</Wix>
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Wix xmlns="http://wixtoolset.org/schemas/v4/wxs" xmlns:util="http://wixtoolset.org/schemas/v4/wxs/util" xmlns:ui="http://wixtoolset.org/schemas/v4/wxs/ui">
+  <?include $(sys.CURRENTDIR)\Workdir\$(var.OutputBaseFilename)-$(var.ProductSku).Variables.wxi ?>
+
+  <Package Name="$(var.ProductNameWithVersion)" Language="$(var.ProductLanguage)" Version="$(var.ProductVersion)" Manufacturer="$(var.ProductManufacturer)" UpgradeCode="$(var.ProductUpgradeCode)" InstallerVersion="500" ProductCode="$(var.ProductId)" Scope="perUserOrMachine"> <!-- This scope sets ALLUSERS to 2 -->
+    <SummaryInformation Description="$(var.PackageDescription)" Manufacturer="$(var.ProductManufacturer)" />
+    <MajorUpgrade DowngradeErrorMessage="!(loc.DowngradeDenied)" />
+    <Media Id="1" Cabinet="Data1.cab" EmbedCab="yes" CompressionLevel="high" />
+    <Property Id="ARPHELPLINK" Value="!(loc.ProductHelpLink)" />
+    <Property Id="ARPURLINFOABOUT" Value="!(loc.ProductUrlInfoAbout)" />
+    <Property Id="ARPURLUPDATEINFO" Value="!(loc.ProductUrlUpdateInfo)" />
+    <Property Id="ARPPRODUCTICON" Value="logo.ico" />
+    <SetProperty Id="ARPINSTALLLOCATION" Value="[INSTALLDIR]" After="CostFinalize" />
+    <Icon Id="logo.ico" SourceFile="{vendor_branding_logo}" />
+
+    <!-- Required for Windows 7 and later. -->
+    <Launch Condition="Installed OR VersionNT &gt;= 601" Message="!(loc.OSVersionRequired)" />
+    <Launch Condition="(Not ITANIUM = &quot;Itanium&quot;) AND (Not ITANIUM = &quot;Itanium 2&quot;)" Message="!(loc.ProductIsNotSupportedOnItanium)" />
+
+    <!--
+      We may need to be able to install JDK in parallel. Therefore an in-place
+      upgrade need to be disabled. Keep the code as reference in case we allow
+      this in future or with JRE.
+      <Upgrade Id="$(var.ProductUpgradeCode)">
+        <UpgradeVersion OnlyDetect="yes" Minimum="$(var.ProductVersion)" Property="NEWPRODUCTFOUND" IncludeMinimum="no" />
+        <UpgradeVersion Minimum="$(var.RTMProductVersion)" IncludeMinimum="yes" Maximum="$(var.ProductVersion)" Property="UPGRADEFOUND" IncludeMaximum="no" MigrateFeatures="yes" />
+      </Upgrade>
+      <CustomAction Id="PreventDowngrading" Error="!(loc.NewerVersionInstalled)" />
+
+      <InstallExecuteSequence>
+        <Custom Action="PreventDowngrading" After="FindRelatedProducts">NEWPRODUCTFOUND</Custom>
+        <RemoveExistingProducts After="InstallFinalize" />
+      </InstallExecuteSequence>
+      <InstallUISequence>
+        <Custom Action="PreventDowngrading" After="FindRelatedProducts">NEWPRODUCTFOUND</Custom>
+      </InstallUISequence>
+    -->
+
+    <!-- Define the directory structure -->

```

Comment:
Guessing you need to move this comment to line up with it's new position at https://github.com/adoptium/installer/pull/851/files#diff-1a76603c3a0aced9dc527a38408ba3816cf0d48cb10487bdbe10dbddad0623a3R211

Reply:
There doesn't seem to be a line number attahed to this link. If I had to guess, maybe you are asking for the comment on line 39 to be moved to line 19?